langchain/vectorstores/pgvector.py

```diff
@@ -593,6 +594,9 @@ def connection_string_from_db_params(
         database: str,
         user: str,
         password: str,
+        **kwargs
```

Comment:
should we just call this query_params and make it an optional dict if that's what it's used for

Reply:
That's a great idea. I'll push an updated PR